storybook/stories/API/cartesian/Bar.stories.tsx

```diff
@@ -0,0 +1,333 @@
+import React, { useState } from 'react';
+import { ComposedChart, Bar, ResponsiveContainer, Cell } from '../../../../src';
+import { coordinateWithValueData, pageData } from '../../data';
+
+export default {
+  component: Bar,
+  argTypes: {
+    background: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    stroke: {
+      control: { type: 'color' },
+    },
+    fill: {
+      control: { type: 'color' },
+    },
+    animationEasing: {
+      options: ['ease', 'ease-in', 'ease-out', 'ease-in-out', 'linear'],
+      control: {
+        type: 'select',
+      },
+    },
+  },
+};
+
+const [surfaceWidth, surfaceHeight] = [600, 300];
+
+const Basic = {
+  render: (args: Record<string, any>) => {
+    const { data, defs, ...areaArgs } = args;
+
+    return (
+      <ResponsiveContainer width="100%" height={surfaceHeight}>
+        <ComposedChart
+          width={surfaceWidth}
+          height={surfaceHeight}
+          margin={{
+            top: 20,
+            right: 20,
+            bottom: 20,
+            left: 20,
+          }}
+          data={data}
+        >
+          {defs}
+          <Bar dataKey="y" isAnimationActive={false} {...areaArgs} />
+        </ComposedChart>
+      </ResponsiveContainer>
+    );
+  },
+  args: {
+    data: coordinateWithValueData,
+    dataKey: 'y',
+  },
+  parameters: { controls: { include: ['data'] } },
```

Comment:
Thinking about this again, I suggest to take out `data` from the controls, because really, data is a property of the ComposedChart. 

Instead I suggest to hardcode a single dataset across all stories. Preferably pageData as used in most examples.